E2E TestUpdateESSecureSettings is failing with 7.2/7.3

```
21:00:12      --- FAIL: TestUpdateESSecureSettings/ES_pods_should_eventually_be_ready (300.00s)
21:00:12          require.go:794: 
21:00:12                  Error Trace:    utils.go:80
21:00:12                  Error:          Received unexpected error:
21:00:12                                  pod test-es-keystore-6h8c-es-masterdata-0 is not ready yet
21:00:12                  Test:           TestUpdateESSecureSettings/ES_pods_should_eventually_be_ready
```

Log:
```
Exception in thread "main" java.lang.IllegalArgumentException: Setting [path.logs] is a non-secure setting and must be stored inside elasticsearch.yml, but was found inside the Elasticsearch keystore
    at org.elasticsearch.common.settings.Setting.innerGetRaw(Setting.java:492)
    at org.elasticsearch.common.settings.Setting.getRaw(Setting.java:479)
    at org.elasticsearch.common.settings.Setting.get(Setting.java:430)
```

According to https://www.elastic.co/guide/en/elasticsearch/reference/current/secure-settings.html:

> Important: 
Only some settings are designed to be read from the keystore. However, the keystore has no validation to block unsupported settings. Adding unsupported settings to the keystore will cause Elasticsearch Additional unsupported settings being added to the keystore will cause Elasticsearch to fail to start. See documentation for each setting to see if it is supported as part of the keystore.

--- a/operators/test/e2e/es/keystore_test.go
+++ b/operators/test/e2e/es/keystore_test.go
@@ -21,6 +21,8 @@ func TestUpdateESSecureSettings(t *testing.T) {
 
 	// user-provided secure settings secret
 	secureSettingsSecretName := "secure-settings-secret"
+	const securePasswordSettingKey = "xpack.notification.email.account.foo.smtp.secure_password"
+	const securetUserSettingKey = "xpack.notification.jira.account.bar.secure_user"
 	secureSettings := corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      secureSettingsSecretName,
@@ -28,8 +30,8 @@ func TestUpdateESSecureSettings(t *testing.T) {
 		},
 		Data: map[string][]byte{
 			// this needs to be a valid configuration item, otherwise ES refuses to start
-			"path.logs":                    []byte("/tmp/logs"),
-			"xpack.security.audit.enabled": []byte("false"),
+			securePasswordSettingKey: []byte("foo_pw"),
+			securetUserSettingKey:    []byte("bar_user"),
 		},
 	}
 
@@ -58,8 +60,8 @@ func TestUpdateESSecureSettings(t *testing.T) {
 		WithSteps(test.StepList{
 			// initial secure settings should be there in all nodes keystore
 			elasticsearch.CheckESKeystoreEntries(k, b.Elasticsearch, []string{
-				"path.logs",
-				"xpack.security.audit.enabled"}),
+				securePasswordSettingKey,
+				securetUserSettingKey}),
 
 			// modify the secure settings secret
 			test.Step{
@@ -67,7 +69,7 @@ func TestUpdateESSecureSettings(t *testing.T) {
 				Test: func(t *testing.T) {
 					// remove some keys, add new ones
 					secureSettings.Data = map[string][]byte{
-						"path.logs": []byte("/tmp/logs2"), // the actual value update cannot be checked :(
+						securePasswordSettingKey: []byte("baz"), // the actual value update cannot be checked :(
 					}
 					err := k.Client.Update(&secureSettings)
 					require.NoError(t, err)
@@ -76,7 +78,7 @@ func TestUpdateESSecureSettings(t *testing.T) {
 
 			// keystore should be updated accordingly
 			elasticsearch.CheckESKeystoreEntries(k, b.Elasticsearch, []string{
-				"path.logs"}),
+				securePasswordSettingKey}),
 
 			// remove the secure settings reference
 			test.Step{

